test/Amadevus.RecordGenerator.Test/RecordMutatorTests.cs

```diff
@@ -32,7 +32,14 @@ public void With_SimpleProperty_CreatesModifiedInstance()
         public void Ctor_InvokesValidate_Throwing()
         {
             var item = CreateItem();
-            Assert.Throws<ArgumentNullException>("value", () => item.WithName(null));
+            //
+            // NOTE! We test that validation is indeed invoked but don't
+            // test the parameter name since it comes from the validation
+            // method that assumes the name of the corresponding parameter on
+            // the constructor ("name") as opposed to WithName (the parameter
+            // of all with-methods is always named "value").
+            //
+            Assert.Throws<ArgumentNullException>(() => item.WithName(null));
```

Comment:
Let's assume it's `nameof(Item.Name)`.

Reply:
Done with 1ff6ba45e45245343dcc73510986db2dd5f976ba.